### ISSUE TITLE
Ads have not been loading for the past few weeks

### ISSUE THREAD

**Issue by cddceo** [ammarahm-ed/react-native-admob-native-ads#391]
It’s not that an error is occurring, but it seems that AdMob ads are not being properly fetched. Is there anyone else experiencing the same issue? If there is a solution, please let me know.

**Comment by JoaquinManeroJazusoft** [comment]
Yes, I have the same problem.  Sometimes they charge and sometimes they don't, what could be the reason? Also, could it be that custom targeting doesn't work? I need to know if it is my problem or the library currently does not work with custom targeting
